### DISCUSSION TITLE
Adding a Custom Emblem

### DISCUSSION THREAD

**Discussion by KazFK1** [sheltermanager/asm3#1444] [Q&A / Help]
I want to add a custom emblem in System Options under Animal Emblems.  I can add the picture, Select "if animal has" but then in the next dropdown box there is nothing to select and I'm not sure where to add selection options.

**Comment by Tetzor** [comment]
Hi,

You need to first create an animal flag (Settings->Lookup data->Animal flags). You will then be able to select that flag in the drop down next to "if animal has".  Any animals you then add that flag to will display your emblem.

Jon
